redbot/cogs/trivia/trivia.py

```diff
@@ -402,6 +405,64 @@ async def trivia_list(self, ctx: commands.Context):
             else:
                 await ctx.send(msg)
 
+    @trivia.command(name="info")
+    async def trivia_info(self, ctx: commands.Context, category: str.lower):
+        """Get information about a trivia category."""
+        try:
+            data = self.get_trivia_list(category)
+        except FileNotFoundError:
+            return await ctx.send(
+                _(
+                    "Invalid category `{name}`. See `{prefix}trivia list` for a list of "
+                    "trivia categories."
+                ).format(name=category, prefix=ctx.clean_prefix)
+            )
+        except InvalidListError:
+            return await ctx.send(
+                _(
+                    "There was an error parsing the trivia list for the `{name}` category. It "
+                    "may be formatted incorrectly."
+                ).format(name=category)
+            )
+
+        default_config = await self.config.guild(ctx.guild).all()
+        if "CONFIG" in data:
+            config_settings = {}
+            config = data.pop("CONFIG")
+            for setting in default_config:
+                config_settings[setting] = config.get(setting, default_config[setting])
+        else:
+            config_settings = default_config
```

Comment:
I do wonder if we really should be showing the whole configuration rather than just the actual overrides.

Reply:
> I do wonder if we really should be showing the whole configuration rather than just the actual overrides.

I suppose " (default)" or something similar could be appended to the value of each setting if the setting remains the same as the default setting

Reply:
I suppose. But is there much benefit to showing the configuration that hasn't changed? I guess it's true that regular users can't use `[p]triviaset showsettings` to see the default configuration but does it matter?

Reply:
I guess not, they'd just remain unaware of any underlying setting changes in the guild's config that override trivia defaults. Though I don't really see that as a problem myself